<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>环世界 Mod 管理器</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="unloadedModsPanel" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="unloadedModsLabel">
          <property name="text">
           <string>未加载的 Mod</string>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <bold>true</bold>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="unloadedSearchEdit">
          <property name="placeholderText">
           <string>搜索类型、名称、备注</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="unloadedModsList">
          <property name="selectionMode">
           <enum>QAbstractItemView::ExtendedSelection</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="addSelectedButton">
          <property name="text">
           <string>添加选中 &gt;&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="loadedModsPanel" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="loadedHeaderLayout">
          <item>
           <widget class="QLabel" name="loadedModsLabel">
            <property name="text">
             <string>已加载的 Mod</string>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
              <bold>true</bold>
             </font>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="autoSortButton">
            <property name="text">
             <string>智能排序 (Ctrl+R)</string>
            </property>
            <property name="toolTip">
             <string>根据依赖关系和类型优先级自动排序</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLineEdit" name="loadedSearchEdit">
          <property name="placeholderText">
           <string>搜索类型、名称、备注</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="loadedModsList">
          <property name="dragDropMode">
           <enum>QAbstractItemView::InternalMove</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="moveUpButton">
            <property name="text">
             <string>上移 ↑</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="moveDownButton">
            <property name="text">
             <string>下移 ↓</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="removeButton">
            <property name="text">
             <string>&lt;&lt; 移除</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="detailPanel" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="detailLabel">
          <property name="text">
           <string>Mod 详情</string>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <bold>true</bold>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QScrollArea" name="scrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>376</width>
             <height>558</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionSaveToGame"/>
    <addaction name="actionSaveAs"/>
    <addaction name="actionLoadConfig"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuManage">
    <property name="title">
     <string>管理</string>
    </property>
    <addaction name="actionManageTypes"/>
    <addaction name="actionTypePriority"/>
    <addaction name="separator"/>
    <addaction name="actionAutoSort"/>
    <addaction name="separator"/>
    <addaction name="actionRescan"/>
    <addaction name="separator"/>
    <addaction name="actionPathSettings"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuManage"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSaveToGame">
   <property name="text">
    <string>保存到游戏配置</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>另存为...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionLoadConfig">
   <property name="text">
    <string>加载配置...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actionManageTypes">
   <property name="text">
    <string>管理 Mod 类型...</string>
   </property>
  </action>
  <action name="actionRescan">
   <property name="text">
    <string>重新扫描 Mod</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>关于...</string>
   </property>
  </action>
  <action name="actionPathSettings">
   <property name="text">
    <string>路径设置...</string>
   </property>
  </action>
  <action name="actionTypePriority">
   <property name="text">
    <string>类型优先级设置...</string>
   </property>
  </action>
  <action name="actionAutoSort">
   <property name="text">
    <string>智能排序</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
